From a13d38ea605a8db7392f90ae1a894e14d7746e8d Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Wed, 9 Dec 2020 19:33:08 +0100
Subject: [PATCH] Emit XorgDisplayServer::started only when the auth file is
 ready

---
 src/daemon/XorgDisplayServer.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/daemon/XorgDisplayServer.cpp b/src/daemon/XorgDisplayServer.cpp
index 5f93a1b..4c51304 100644
--- a/src/daemon/XorgDisplayServer.cpp
+++ b/src/daemon/XorgDisplayServer.cpp
@@ -222,8 +222,6 @@ namespace SDDM {
 
             // close our pipe
             close(pipeFds[0]);
-
-            emit started();
         }
 
         // The file is also used by the greeter, which does care about the
@@ -236,6 +234,8 @@ namespace SDDM {
         }
         changeOwner(m_authPath);
 
+        emit started();
+
         // set flag
         m_started = true;
 
-- 
2.35.1

