From 7887053b8ef39f233ab0e219c6e8518cf1bcfa4d Mon Sep 17 00:00:00 2001
From: Nathan <ndowens@artixlinux.org>
Date: Fri, 18 Nov 2022 20:32:10 -0600
Subject: [PATCH] artix

---
 kde-modules/KDEInstallDirsCommon.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kde-modules/KDEInstallDirsCommon.cmake b/kde-modules/KDEInstallDirsCommon.cmake
index cc8b792..b519640 100644
--- a/kde-modules/KDEInstallDirsCommon.cmake
+++ b/kde-modules/KDEInstallDirsCommon.cmake
@@ -27,7 +27,7 @@ set(_LIBDIR_DEFAULT "lib")
 # See https://wiki.debian.org/Multiarch
 if((CMAKE_SYSTEM_NAME MATCHES "Linux|kFreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "GNU")
    AND NOT CMAKE_CROSSCOMPILING
-   AND NOT EXISTS "/etc/arch-release"
+   AND NOT EXISTS "/etc/artix-release"
    AND NOT DEFINED ENV{FLATPAK_ID}
    AND NOT EXISTS "/etc/NIXOS")
   if (EXISTS "/etc/debian_version") # is this a debian system ?
-- 
2.38.1

