diff -Naur cinnamon-desktop-5.2.0-orig/schemas/org.cinnamon.desktop.background.gschema.xml.in cinnamon-desktop-5.2.0/schemas/org.cinnamon.desktop.background.gschema.xml.in
--- cinnamon-desktop-5.2.0-orig/schemas/org.cinnamon.desktop.background.gschema.xml.in	2021-11-15 09:11:43.000000000 -0300
+++ cinnamon-desktop-5.2.0/schemas/org.cinnamon.desktop.background.gschema.xml.in	2021-12-17 16:24:13.483229140 -0300
@@ -11,7 +11,7 @@
       </description>
     </key>
     <key name="picture-uri" type="s">
-      <default>'file://@datadir@/themes/Adwaita/backgrounds/adwaita-timed.xml'</default>
+      <default>'file://@datadir@/backgrounds/gnome/adwaita-timed.xml'</default>
       <summary>Picture URI</summary>
       <description>
         URI to use for the background image. Not that the backend only supports
diff -Naur cinnamon-desktop-5.2.0-orig/schemas/org.cinnamon.desktop.interface.gschema.xml.in cinnamon-desktop-5.2.0/schemas/org.cinnamon.desktop.interface.gschema.xml.in
--- cinnamon-desktop-5.2.0-orig/schemas/org.cinnamon.desktop.interface.gschema.xml.in	2021-11-15 09:11:43.000000000 -0300
+++ cinnamon-desktop-5.2.0/schemas/org.cinnamon.desktop.interface.gschema.xml.in	2021-12-17 16:24:44.776561640 -0300
@@ -98,7 +98,7 @@
       </description>
     </key>
     <key name="icon-theme" type="s">
-      <default>'gnome'</default>
+      <default>'Adwaita'</default>
       <summary>Icon Theme</summary>
       <description>
         Icon theme to use for the panel, nautilus etc.
