From c6556426c9ee5be3eebbb278536cbf86ca6643a1 Mon Sep 17 00:00:00 2001
From: Nathan <ndowens@artixlinux.org>
Date: Wed, 25 Jan 2023 05:17:55 -0600
Subject: [PATCH] quazip-1.4

---
 cmake/modules/FindQUAZIP.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/modules/FindQUAZIP.cmake b/cmake/modules/FindQUAZIP.cmake
index 512803e..f0118c1 100644
--- a/cmake/modules/FindQUAZIP.cmake
+++ b/cmake/modules/FindQUAZIP.cmake
@@ -17,7 +17,7 @@ ELSEIF (QUAZIP_INCLUDE_DIRS AND QUAZIP_LIBRARIES)
 	SET(QUAZIP_FOUND TRUE)
 ELSE ()
     IF (Qt5Core_FOUND)
-        set(QUAZIP_LIB_VERSION_SUFFIX 5)
+        set(QUAZIP_LIB_VERSION_SUFFIX Qt5)
     ENDIF()
 	IF (WIN32)
 		FIND_PATH(QUAZIP_LIBRARY_DIR
@@ -35,12 +35,12 @@ ELSE ()
 		pkg_check_modules(PC_QUAZIP quazip)
 		FIND_LIBRARY(QUAZIP_LIBRARIES
 			WIN32_DEBUG_POSTFIX d
-            NAMES quazip${QUAZIP_LIB_VERSION_SUFFIX}
+            NAMES quazip${QUAZIP_LIB_VERSION_SUFFIX} libquazip1-qt5
 			HINTS /usr/lib /usr/lib64
 		)
 		FIND_PATH(QUAZIP_INCLUDE_DIR quazip.h
 			HINTS /usr/include /usr/local/include
-			PATH_SUFFIXES quazip${QUAZIP_LIB_VERSION_SUFFIX}
+			PATH_SUFFIXES quazip${QUAZIP_LIB_VERSION_SUFFIX} QuaZip-Qt5
 		)
 		FIND_PATH(QUAZIP_ZLIB_INCLUDE_DIR zlib.h HINTS /usr/include /usr/local/include)
 	ENDIF (WIN32)
-- 
2.39.1

