--- a/kde-modules/KDEInstallDirsCommon.cmake	2021-12-13 11:04:07.524555495 +0100
+++ b/kde-modules/KDEInstallDirsCommon.cmake	2021-12-13 11:09:05.213633299 +0100
@@ -26,7 +26,7 @@
 # See https://wiki.debian.org/Multiarch
 if((CMAKE_SYSTEM_NAME MATCHES "Linux|kFreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "GNU")
    AND NOT CMAKE_CROSSCOMPILING
-   AND NOT EXISTS "/etc/arch-release"
+   AND NOT EXISTS "/etc/artix-release"
    AND NOT DEFINED ENV{FLATPAK_ID})
   if (EXISTS "/etc/debian_version") # is this a debian system ?
     if(CMAKE_LIBRARY_ARCHITECTURE)
